content/_data/changelogs/weekly.yml

```diff
@@ -1176,7 +1176,35 @@
           title: notification
         - issue: 47139
       pull: 3050
-
+- version: "2.82"
+  date: 2017-10-01
+  changes:
+    - type: major bug
+      message: >
+        <tt>favicon.ico</tt> and other binary resource files were broken since 2.79 because they were incorrectly filtered during the build.
+      issue: 47127
+      pull: 3052
+    - type: bug
+      message: >
+        Don't log warning when an anonymous user sends an invalid crumb, usually just an expired session.
+      issue: 40344
+      pull: 3049
+    - type: major bug
+      message: >
+        Developer: Fix <code>TimeDuration</code> time unit handling and its incorrect usage.
```

Comment:
From what I see it was a real defect which may be somehow impacting users, not 100% sure

Reply:
@jglick already gave an example somewhere of a possible issue, but well:

If users uses CLI (or UI) and manually put `?delay=3` instead of `?delay=3secs`, then yes it will result in a change in behaviour. Though the default generated link for building jobs is for instance `/build?delay=0sec` so though undocumented, removing the `sec` suffix would have been pretty daring.